applications/crossbar/src/modules/cb_auth.erl

```diff
@@ -150,9 +151,14 @@ authorize(Context, PathToken, Id) ->
 
 %% authorize /auth
 -spec authorize_nouns(cb_context:context(), req_verb(), req_nouns()) -> boolean().
-authorize_nouns(C, ?HTTP_PUT, [{<<"auth">>, _}]) -> cb_context:is_superduper_admin(C);
+authorize_nouns(C, ?HTTP_PUT, [{<<"auth">>, _}]) -> authorize_action(C, cb_context:req_value(C, <<"action">>));
 authorize_nouns(C, ?HTTP_PUT, [{<<"auth">>, _}, {<<"accounts">>, _}]) -> cb_context:is_account_admin(C);
-authorize_nouns(C, ?HTTP_PUT, [{<<"auth">>, _}, {<<"users">>, _}, {<<"accounts">>, _}]) -> cb_context:is_account_admin(C);
+authorize_nouns(C, ?HTTP_PUT, [{<<"auth">>, _}, {<<"users">>, [UserId]}, {<<"accounts">>, [AccountId]}]) ->
```

Comment:
@danielfinke is this needed here ?

Reply:
@lazedo yeah, the change here allows users to change their own signature secrets